core-bundle/contao/dca/tl_content.php

```diff
@@ -859,13 +859,17 @@ public function adjustDcaByType($dc)
 	/**
 	 * Set default element type based on user access.
 	 */
-	public function setDefaultElementType()
+	public function setDefaultElementType(DataContainer $dc)
```

Comment:
This method is never called anywhere, is it?

Reply:
it is an onload_callback (has always been)

Reply:
also see https://github.com/contao/contao/pull/7516/files#diff-59be1b35c50ae8aa4f09cce6f8ee0e87da8481d238a52d7f864a682749f7b1e5L45

Reply:
Sorry 😞